bigquery_etl/public_data/publish_gcs_metadata.py

```diff
@@ -78,7 +82,7 @@ def table_metadata_to_json(self):
             metadata_json["review_link"] = REVIEW_LINK + self.metadata.review_bug()
 
         metadata_json["files_uri"] = self.files_uri
-        # todo: add last updated
+        metadata_json["last_updated"] = self.last_updated_uri
```

Comment:
Should we call the key `last_updated_uri`? I would naively expect a field called `last_updated` to be a timestamp string.